apps/cli/src/commands/device/managerListApps.ts

```diff
@@ -4,6 +4,11 @@ import { withDevice } from "@ledgerhq/live-common/hw/deviceAccess";
 import getDeviceInfo from "@ledgerhq/live-common/hw/getDeviceInfo";
 import { listAppsUseCase } from "@ledgerhq/live-common/device/use-cases/listAppsUseCase";
 import { deviceOpt } from "../../scan";
+import { ListAppsEvent, ListAppsResult } from "@ledgerhq/live-common/apps/types";
+
+function isResultEvent(event: ListAppsEvent): event is { type: "result"; result: ListAppsResult } {
+  return event.type === "result";
+}
```

Comment:
Nice, another small improvement we could do would be to create a type for `{ type: "result"; result: ListAppsResult }` and use it both here and in `ListAppsEvent`